### ISSUE TITLE
[BUG] [Go service&Java client]Failed to get multiple locales' translation when one of locales doesn't has available translation.

### ISSUE THREAD

**Issue by lyiyu66** [vmware/singleton#1941]
**Describe the bug**
Go service: 0db26522c08cbaf4f0af68c699ea5afdc252ce5d
Java client: 1e28f1b5a30885d258b4614928bb99dc11be19a4
Failed to get multiple locales' translation when one of locales doesn't has available translation.

**To Reproduce**
1. Invoke the interface of getting multiple components' translation as below:
```
		// messages_ar.json(not available on service) messages_fr.json(available on service)
                Locale locale2 = new Locale("fr");
                Locale locale4 = new Locale("ar");	
		Map<Locale, Map<String, Map<String, String>>> result = tm.getStrings(
                Stream.of(locale2, locale4).collect(Collectors.toSet()), Sets.newHashSet(component1, component2));
```
2. See error
![image](https://user-images.githubusercontent.com/51813046/201807864-ed75f1c8-342c-4c23-9727-a3677fb9a5dc.png)

**Expected behavior**
No error is occurred.

**Additional context**
Not reproduceable on Java Service.

**Comment by huihuiw01** [comment]
I think the reason is Go Service's response data is not the same with Java Service, and change should be done in Go Service to fix this bug. Please confirm @Xiaochao8 .

**Comment by lyiyu66** [comment]
The issue is fixed with commit a3840c9ffc8abf789b0306105e95ce760115c1c6, no error is occurred.
